### PR TITLE
fix(label-group): truncate labels within label-group

### PR DESCRIPTION
**Pull request by sg00dwin**
fixes: https://github.com/patternfly/patternfly/issues/7104

[Labels with truncation](https://patternfly-pr-7176.surge.sh/components/label#labels-with-truncation)

Screenshot 2024-10-24 at 4 05 04 PM

Screenshot 2024-10-24 at 4 05 19 PM

### CODE REVIEW

File: src/patternfly/components/Label/examples/Label.md [hunk 1]
```diff
@@ -563,6 +563,55 @@ In addition to the JavaScript management of [editable labels](/components/label#
...
+### Labels with truncation
+```hbs
+<div class="pf-v6-c-label-truncate--example">
+<strong>Label</strong>
+<br/><br/>
+{{> label
+label--IsRemovable=true
+label--id=(concat label--variants--id '-truncated-with-icon')
+label-text--value=(concat label--variants--title ' Label with set max-width truncation')
+label-text--attribute='style="--pf-v6-c-label__text--MaxWidth: 38ch"'}}
```
[the remark this answers is not in the document]
Comment (mattnolting) [reply src/patternfly/components/Label/examples/Label.md:575]: NIT: Can you indent these params?

**Review by mattnolting — APPROVED** [review]
LGTM 🤖

File: src/patternfly/components/Label/examples/Label.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (sg00dwin) [reply src/patternfly/components/Label/examples/Label.md]: updated

**Comment by srambach** [comment]
What do you think about a `min-width` somewhere to avoid the labels shrinking to nothing? Maybe something along the lines of this? 
```.pf-v6-c-label {
    min-width: 4em;
}
```
image

**Comment by sg00dwin** [comment]
> What do you think about a `min-width` somewhere to avoid the labels shrinking to nothing? Maybe something along the lines of 4em or so, on the label?

Added a `min-width` of 3 em which matches a 4 character width.

**Pull request merged**